storage/replica_test.go

```diff
@@ -2661,7 +2661,7 @@ func TestRangeStatsComputation(t *testing.T) {
 	if _, err := client.SendWrapped(tc.Sender(), tc.rng.context(), &pArgs); err != nil {
 		t.Fatal(err)
 	}
-	expMS := engine.MVCCStats{LiveBytes: 24, KeyBytes: 14, ValBytes: 10, IntentBytes: 0, LiveCount: 1, KeyCount: 1, ValCount: 1, IntentCount: 0, IntentAge: 0, GCBytesAge: 0, SysBytes: 55, SysCount: 1, LastUpdateNanos: 0}
+	expMS := engine.MVCCStats{LiveBytes: 25, KeyBytes: 14, ValBytes: 11, IntentBytes: 0, LiveCount: 1, KeyCount: 1, ValCount: 1, IntentCount: 0, IntentAge: 0, GCBytesAge: 0, SysBytes: 51, SysCount: 1, LastUpdateNanos: 0}
```

Comment:
The changes to `roachpb.Value` reduces the encoded size for values with checksums, but the values in these tests prior to this change did not have checksums while now they have a 0 (uninitialized checksum). The result is you see an increase in size of 1 byte for some of the stats, but really that would be a decrease in size of 4 bytes if the test had initialized a checksum previously.